src/lib.rs

```diff
@@ -715,18 +726,18 @@ fn check_annotations(
         });
     }
 
-    let filter = |msgs: Vec<Message>| -> Vec<_> {
+    let filter = |msgs: Vec<Message>, errors: &mut Vec<_>| -> Vec<_> {
         msgs.into_iter()
             .filter(|msg| {
                 msg.level
                     >= comments
-                        .require_annotations_for_level
+                        .find_one_for_revision(revision, |r| r.require_annotations_for_level, |_| errors.push(Error::InvalidComment { msg: "`require_annotations_for_level` specified twice for same revision".into(), line: 0 }))
```

Comment:
Can you let-bind the long `comments....` expression here? This is an awfully long line

Reply:
Also I think this can be moved out of the `filter` loop?

Reply:
oh wow rustfmt just completely gave up :D yea I'll clean that up